bootspec/src/deser.rs

```diff
@@ -35,6 +35,14 @@ impl<'de> Visitor<'de> for BootSpecExtensionsVisitor {
             map.insert(key, value);
         }
 
+        for (k, v) in map.iter() {
+            if v.is_null() {
```

Comment:
Note that this doesn't preclude e.g. `"org.test": { "enableSomeSetting": null }` -- only `"org.test": null`